packages/flutter/lib/src/foundation/bitfield.dart

```diff
@@ -3,15 +3,15 @@
 // found in the LICENSE file.
 
 import '_bitfield_io.dart'
-  if (dart.library.html) '_bitfield_web.dart' as _bitfield;
+  if (dart.library.html) '_bitfield_web.dart' as bitfield;
```

Comment:
I would argue that these filenames should probably also be renamed to not start with an "_". That way, they will appear right next to this file when files are sorted alphabetically.

Reply:
I concur. :)

Reply:
I'll send a separate PR for that.